packages/expo-image/ios/ImageModule.swift

```diff
@@ -104,6 +104,22 @@ public final class ImageModule: Module {
         promise.resolve(true)
       }
     }
+
+    AsyncFunction("getCachePathAsync") { (source: String, promise: Promise) in
```

Comment:
What do you think about renaming `source` to `cacheKey`? In the `source` view prop you can customize the cache key to be something different than the `uri`, so I think it would be more precise.